api/src/utils/apply-query.ts

```diff
@@ -539,23 +539,23 @@ export function applyFilter(
 			// Knex supports "raw" in the columnName parameter, but isn't typed as such. Too bad..
 			// See https://github.com/knex/knex/issues/4518 @TODO remove as any once knex is updated
 
-			// These operators don't rely on a value, and can thus be used without one (eg `?filter[field][_null]`)
-			if (operator === '_null' || (operator === '_nnull' && compareValue === false)) {
-				dbQuery[logical].whereNull(selectionRaw);
+			const boolValue = Boolean(compareValue ?? true);
+			if ((operator === '_null' && boolValue) || (operator === '_nnull' && !boolValue)) {
+				return dbQuery[logical].whereNull(selectionRaw);
```

Comment:
```suggestion
				dbQuery[logical].whereNull(selectionRaw);
				return;
```
The return statements here should be on the next line, as this is a `void` function. This same change should be applied to other blocks below.